src/time.cr

```diff
@@ -192,30 +192,34 @@ struct Time
   # time = Time.epoch_ms(981173106789) # => 2001-02-03 04:05:06.789 UTC
   # time.millisecond                   # => 789
   # ```
-  def self.epoch_ms(milliseconds : Int) : self
+  def self.epoch_ms(milliseconds : Int) : Time
     milliseconds = milliseconds.to_i64
     seconds = UNIX_SECONDS + (milliseconds / 1_000)
     nanoseconds = (milliseconds % 1000) * NANOSECONDS_PER_MILLISECOND
     new(seconds: seconds, nanoseconds: nanoseconds.to_i, kind: Kind::Utc)
   end
 
-  def clone
+  def clone : self
     self
   end
 
-  def +(other : Span)
+  # Returns a `Time` that is later by this `Time::Span`.
```

Comment:
`s/this/a/`? hmm.

"later by this timespan" doesn't quite sound right to me.

Reply:
I mean... I've considered at least 4 options and none sound right to me. What do?

Reply:
C# uses `Returns a new DateTime that adds the value of the specified TimeSpan to the value of this instance.`

Reply:
Sure